var/spack/repos/builtin/packages/py-charm4py/package.py

```diff
@@ -36,7 +36,7 @@ class PyCharm4py(PythonPackage):
     depends_on("py-setuptools", type="build")
     # in newer pip versions --install-option does not exist
     depends_on("py-pip@:23.0", type="build")
-    depends_on("py-cython", type="build")
+    depends_on("py-cython@:3.0.0", type="build")
```

Comment:
```suggestion
    depends_on("py-cython@:2", type="build")
```
Version ranges are inclusive in Spack